documentation/behaviors/validate.markdown

```diff
@@ -443,12 +443,12 @@ Done! Now you can use your custom validator constraint in your `schema.xml` file
 <!-- end of your schema -->
 ```
 
-**Note**: if you think your custom constraint could be generic enough to be useful for the community, please submit it to Propel team,
-to include it in Propel bundled constraints (see [http://dotheweb.posterous.com/open-source-is-a-gift](http://dotheweb.posterous.com/open-source-is-a-gift)).
+**Note**: if you think your custom constraint is generic enough for community use, please submit it to the Propel team
+to include it in our bundled constraints (see [http://dotheweb.posterous.com/open-source-is-a-gift](http://www.redotheweb.com/2011/11/13/open-source-is-a-gift.html)).
```

Comment:
The displayed URL isn't  fixed here ; maybe instead of pasting it, you can wrap the URL with chevrons so it will be auto-linked (`<http://www.redotheweb.com/2011/11/13/open-source-is-a-gift.html>`). Nice catch though, broken links are pretty awkward! :+1: